### PR TITLE
Add Portugese translation

### PR DESCRIPTION
**Pull request by glaucia86**
This is a translate for Portuguese. This Pull Request is related with issue: #84 

- [x] - README (from the main)
- [x] - README (session 00)
- [x] - README (session 01)
- [x] - README (session 02)
- [x] - README (session 03)
- [x] - README (session 04)
- [x] - README (session 05)
- [x] - README (session 06)
- [x] - README (session 07)
- [x] - README (session 08)
- [x] - README (session 09)
- [x] - README (session 10)
- [x] - README (session 11)
- [x] - README (session 12)
- [x] - README (session 13)

### CODE REVIEW

**Comment by softchris** [comment]
@PabloNunes please have look :)

**Comment by glaucia86** [comment]
@koreyspace I never saw this: why are you using tracking codes in images and also places that we shouldn't include it?

**Comment by john0isaac** [comment]
This is not an error from your side @glaucia86 there is a file that doesn't exist in the repo.
I have highlighted this problem here. https://github.com/microsoft/generative-ai-for-beginners/issues/155

The checks won't pass unless this file path is fixed.

**Comment by glaucia86** [comment]
> This is not an error from your side @glaucia86 there is a file that doesn't exist in the repo. I have highlighted this problem here. #155
> 
> The checks won't pass unless this file path is fixed.

Thanks for the warning, @john0isaac

**Comment by leestott** [comment]
@PabloNunes  can you please take a look at this PR for translation correctness.

**Review by skydiver1980 — APPROVED** [review]

**Review by PabloNunes — APPROVED** [review]
**Review translation PT-BR**

- Adding translation to PT-BR
- Fixing links and some terms in the current content as well

File: 00-course-setup/translations/pt-br/README.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+### 1. Faça um Fork deste Repositório
+
+[Faça um fork deste repositório](https://github.com/microsoft/generative-ai-for-beginners/fork?WT.mc_id=academic-105485-koreyst) para a sua própria conta no GitHub para que possa alterar qualquer código e concluir os desafios. Você também pode [marcar com uma (🌟) este repositório](https://docs.github.com/en/get-started/exploring-projects-on-github/saving-repositories-with-stars?WT.mc_id=academic-105485-koreyst) para encontrar com mais facilidade esse repositório.
+
+### 2. Crie um Codespaces
+
+Para evitar problemas de dependência ao executar o código, recomendamos a execução deste curso em um Codespaces do GitHub.
+
+Isso pode ser criado selecionando a opção `Code` na sua versão `birfucada` deste repositório e selecionando a opção **Codespaces**.
```
[the remark this answers is not in the document]
Comment (glaucia86) [reply 00-course-setup/translations/pt-br/README.md:19]: Vou alterar aqui

**Pull request merged**